source/fundamentals/collations.txt

```diff
@@ -4,5 +4,311 @@ Collations
+
+   myCollation := &options.Collation{Locale: "fr", Strength: 1}
+   opts := options.CreateCollection().SetCollation(myCollation)
+   err := db.CreateCollection(context.TODO(), "books", opts)
+
+   if err != nil {
+      log.Fatal(err)
+   }
+
+Use the Default Collation Example
+`````````````````````````````````
+
+If you call an operation that supports collation on the ``books`` collection, the operation
+will now use the default collation.
+
+Assume the ``books`` collection contains the following documents:
+
+.. code-block:: json
+
+   {"name" : "Emma", "length" : "474"}
+   {"name" : "Les Misérables", "length": "1462"}
+   {"name" : "Infinite Jest", "length" : "1104"}
+   {"name" : "Cryptonomicon", "length" : "918"}
+   {"name" : "Ça", "length" : "1138"}
+
+.. note::
+
+   To learn how to insert documents, see :ref:`golang-insert-guide`.
+   
+You can use an operation with the default collation on the ``books`` collection specified
+in the :ref:`golang-create-collection`. The following example uses the ``Find()`` method
+to return all documents with a ``name`` value that alphabetically precedes ``"Infinite Jest"``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: go
+
+      filter := bson.D{{"name", bson.D{{"$lt", "Infinite Jest"}}}}
+      cursor, err := coll.Find(context.TODO(), filter)
+
+      var results []bson.D
+      if err = cursor.All(context.TODO(), &results); err != nil {
+         panic(err)
+      }
+      for _, result := range results {
+         fmt.Println(result)
+      }
+
+   .. output::
+      :language: none
+      :visible: false
+
+      [{name Ça} {length 1138}]
+      [{name Cryptonomicon} {length 918}]
+      [{name Emma} {length 474}]
+
+Without specifying a default ``books`` collation, the ``Find()`` method would follow default
+binary collation rules to determine the ``name`` values that precede ``"Infinite Jest"``. These
+rules place words beginning with "Ç" after those beginning with "I". The output would resemble
+the following:
+
+.. code-block:: json
+   :copyable: false
+
+   [{name Cryptonomicon} {length 918}]
+   [{name Emma} {length 474}]
+
+See :ref:`golang-retrieve` for help with using the ``Find()`` method.
+
+.. _golang-index-collation:
+
+Set a Collation on an Index 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can apply a collation when you create a new index on a collection. The index stores
+an ordered representation of the documents in the collection so your MongoDB instance
+does not need to perform the ordering in-memory. 
+
+To use the index in an operation, your operation must use the same collation as the one
+specified in the index. Additionally, ensure that the operation is covered by the index that
+contains the collation. Set a collation through an ``IndexOptions`` object and call the
+``CreateOne()`` method with your options object as an argument.
+
+Example
+```````
+
+After creating the ``books`` collection and applying a default collation, you cannot change
```

Comment:
Issue: As a reader, if I went directly to the ``Set a Collation on an Index`` section from the righthand TOC, I wouldn't know what this was referring to.

Suggestion: Add a hyper link to where on the page this collection is created.

Reply:
Made this change